string switch_command
string switch_name
int8 switch_number
bool biais_state
int64 biais_id
string biais_info